renku/api/storage.py

```diff
@@ -88,24 +88,30 @@ def track_paths_in_storage(self, *paths):
                 stderr=STDOUT,
                 cwd=str(self.path),
             )
-        elif self.use_external_storage:
+        else:
             raise errors.ExternalStorageNotInstalled(self.repo)
 
     def untrack_paths_from_storage(self, *paths):
         """Untrack paths from the external storage."""
-        if self.use_external_storage and self.external_storage_installed:
+        if not self._use_lfs():
+            return
+
+        if self.external_storage_installed:
             call(
                 self._CMD_STORAGE_UNTRACK + list(paths),
                 stdout=PIPE,
                 stderr=STDOUT,
                 cwd=str(self.path),
             )
-        elif self.use_external_storage:
+        else:
```

Comment:
Instead of adding else, we could check for the oppositte case.

```
if not self.external_storage_installed:
    raise errors.ExternalStorageNotInstalled(self.repo)
```